public/service_portal/v0.1.0/openapi.json

```diff
@@ -967,6 +967,36 @@
                         "readOnly": true,
                         "title": "State",
                         "example": "The current state of a portfolio item"
+                    },
+                    "display_name": {
+                        "type": "string",
+                        "readOnly": true,
+                        "title": "Display Name",
+                        "example": "The name of the portfolio item to be displayed in the UI"
+                    },
+                    "long_description": {
+                        "type": "string",
+                        "readOnly": true,
+                        "title": "Long Description",
+                        "example": "The longer description of a portfolio item"
+                    },
+                    "provider_display_name": {
+                        "type": "string",
+                        "readOnly": true,
+                        "title": "Provider Display Name",
+                        "example": "The name of the provider to be dipslayed in the UI"
+                    },
+                    "documentation_url": {
+                        "type": "string",
+                        "readOnly": true,
+                        "title": "Documentation URL",
+                        "example": "The URL for documentation of the portfolio item"
+                    },
+                    "support_url": {
+                        "type": "string",
```

Comment:
@lindgrenj6 - Do we also need the icon here? I realize it's a lookup but it is an attribute off of `PortfolioItem` correct?

Reply:
Yes - I added the icon field to my other PR that has the rest of the icon stuff.